mlir/lib/Dialect/SparseTensor/Transforms/Utils/SparseTensorLevels.h

```diff
@@ -0,0 +1,109 @@
+//===- SparseTensorLevels.h -------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_DIALECT_SPARSETENSOR_TRANSFORMS_UTILS_SPARSETENSORLEVEL_H_
```

Comment:
I think you will need to rename the files

SparseTensorLevels.* -> SparseTensorLevel.*

without the "s", or otherwise, update the class name and macros